app/interactors/content/update_interactor.rb

```diff
@@ -10,8 +10,8 @@ class Content::UpdateInteractor < ApplicationInteractor
   def call
     Edition.transaction do
       find_and_lock_edition
-      update_revision
       check_for_issues
+      update_revision
 
       update_edition
```

Comment:
Probably for another day but I wonder if we want to combine the update edition and revision steps in interactors where they follow each other.

Reply:
That occurred to me too, and it would not be possible to do so. Some of the `update_revision` or `update_edition` methods can be quite long, though. Having them separate has the small advantage for me of explicitly articulating the steps we take for the revision, and the steps we take for the edition.